### DISCUSSION TITLE
VSCode new feature: nest *.meta file within the file

### DISCUSSION THREAD

**Discussion by linonetwo** [TiddlyWiki/TiddlyWiki5#6680] [General]
search for `nesting` in the config, add new rules, and open it
![截屏2022-05-06 16 21 23](https://user-images.githubusercontent.com/3746270/167095007-79597f4d-ace5-4d13-b639-8d7e3a6ab03c.png)

**Comment by pmario** [comment]
What should us tell this info?

**Reply by linonetwo** [reply]
I use vscode to develop plugin, so this is convenient for me.

**Reply by pmario** [reply]
That's nice. But what does it do?

**Reply by Jermolene** [reply]
@pmario the configuration change suggested by @linonetwo  causes `.meta` files to be displayed in the file list with a disclosure arrow underneath the file to which they relate. It may not make sense if you are not a VS Code user.

The illustration kindly provided by @linonetwo is helpful: on the left it shows you how to change the setting, and on the right it shows you the effect in the file browser.

**Reply by linonetwo** [reply]
Yes, you can also read it on vscode release note https://code.visualstudio.com/updates/v1_67#_explorer-file-nesting

**Reply by pmario** [reply]
Thx. I found it reading the release notes. ...

**Comment by joshuafontany** [comment]
Nice feature, thanks @linonetwo !

**Reply by Jermolene** [reply]
@joshuafontany is it something that could be added to your TW5 plugin?

**Reply by pmario** [reply]
@Jermolene ... I think it's a VSCode setting that activates it. So no "plugin" needed.

**Reply by Jermolene** [reply]
> @Jermolene ... I think it's a VSCode setting that activates it. So no "plugin" needed.

I know that, but my understanding is that VSCode plugins can include settings changes, which would mean that end users wouldn't have to hunt around for the setting, but just include the TiddlyWiki plugin.

**Reply by joshuafontany** [reply]
That is an excellent idea. I will look into it.
